### ISSUE TITLE
Error on reorder

### ISSUE THREAD

**Issue by asadkhanmaharvi** [YifanDengWHU/DDIMDL#3]
i have install all the packages according to your packages version but still facing error on reorder ?

**Comment by YifanDengWHU** [comment]
Hi! I just checked for a while. I found that scikit-learn with 0.21.2 can work with the reorder parameter. Could you please provide 
 your pip list and the errors you encounter with?

**Comment by asadkhanmaharvi** [comment]
Yes it is fine for 0.21.2, thank you one thin more that how can i get accuracy score exact that you have achieved ?

**Comment by YifanDengWHU** [comment]
There is uncertainty due to the property of neural network.  So the completely same score is hard to obtain. But using the same feature combination (SMILES+target+enzyme) and the same parameter setting, I think you can get similar results. Try to run more times.

**Comment by asadkhanmaharvi** [comment]
Thank you, I would like to completely understand it I'll be glad if you help me in some terms.
